raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -996,18 +1003,28 @@ def mediate_transfer(
     send a refund back to the payer, allowing the payer to try a different
     route.
     """
-    reachable_routes = filter_reachable_routes(possible_routes, nodeaddresses_to_networkstates)
-    available_routes = filter_used_routes(state.transfers_pair, reachable_routes)
-
     assert payer_channel.partner_state.address == payer_transfer.balance_proof.sender
 
-    transfer_pair, mediated_events = forward_transfer_pair(
-        payer_transfer,
-        available_routes,
-        channelidentifiers_to_channels,
-        pseudo_random_generator,
-        block_number,
-    )
+    transfer_pair: Optional[MediationPairState] = None
+    mediated_events: List[Event] = list()
+
+    reachable_route_states = [
```

Comment:
What you've done here is remove a call to `filter_reachable_routes` which originally filtered the routes according to the availability of the the nodes and manually did the same thing here. Let's please maintain one function which is preferrable the one you removed and do the filtering there.